pkg/datapath/iptables/iptables.go

```diff
@@ -234,6 +234,18 @@ func (m *IptablesManager) removeCiliumRules(table string, prog iptablesInterface
 			continue
 		}
 
+		// Temporary fix while Iptables is upgraded to >= 1.8.5
+		// (See GH-20884).
+		//
+		// The version currently shipped with Cilium (1.8.4) does not
+		// support the deletion of NOTRACK rules, so we will just ignore
+		// them here and let the agent remove them when it deletes the
+		// entire chain.
+		if strings.Contains(rule, "-j NOTRACK") ||
+			strings.Contains(rule, "-j CT --notrack") {
```

Comment:
Doesn't the `-j CT --notrack` work on < 1.8.5?

Reply:
I think I tested restarting a 1.11 agent (which is using `-j CT --notrack`) and it failed to delete the old rules, but I might be mixing things up at this point, let me rerun the test

Reply:
You are right, no need to do anything with `-j CT --notrack` rules :+1: